ledger/sandbox/src/main/scala/platform/sandboxnext/Runner.scala

```diff
@@ -206,6 +206,8 @@ class Runner(config: SandboxConfig) extends ResourceOwner[Port] {
                     else IndexerStartupMode.MigrateAndStart,
                   eventsPageSize = config.eventsPageSize,
                   allowExistingSchema = true,
+                  enableAppendOnlySchema = config.enableAppendOnlySchema,
+                  enableCompression = config.enableCompression,
```

Comment:
See, this is why I don't like default parameters.

Can we remove the default values from these and force callers to make a choice?